src/applications/mhv-medications/actions/prescriptions.js

```diff
@@ -108,3 +108,7 @@ export const fillPrescriptions = prescriptions => async dispatch => {
     return error;
   }
 };
+
+export const clearFillNotification = async dispatch => {
```

Comment:
would you be able to add a test case for this function?

Reply:
TY. done!